server/davidia/models/messages.py

```diff
@@ -224,6 +231,26 @@ def ml_data_is_not_empty(cls, v):
             return v
         raise ValueError("ml_data contains no LineData", v)
 
+    @field_validator("ml_data")
+    @classmethod
+    def line_keys_are_unique(cls, v):
```

Comment:
Not needed if you use a UUID for the key. Instead write a helper function to auto-name lines if they don't have one